src/main/java/sdu/sem2/se17/persistence/data/UserHandlerImpl.java

```diff
@@ -15,26 +21,135 @@ public UserHandlerImpl(DataSource dataSource) {
 
     @Override
     public Optional<User> create(User user) {
-        return Optional.empty();
+         Optional<User> result = Optional.empty();
+        try (
+                Connection connection = dataSource.getConnection();
+                PreparedStatement statement = connection.prepareStatement(""" 
+                    INSERT INTO User (username, email, password, usertype)
+                    VALUES (?, ?, ?, ?)
```

Comment:
Enums must be specified 
```suggestion
                    VALUES (?, ?, ?, ?::userrole)
```